subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/CreatePullRequest.java

```diff
@@ -37,16 +43,22 @@ public void createPullRequest(CreatePullRequestTask task, GitHubApi gitHubApi) t
         gitHubApi.post("/repos/" + task.getUpstreamRepositoryName() + "/pulls", body);
     }
 
+    private void checkPullRequestMetadata(CreatePullRequestTask task) {
+        if (isBlank(getTitle(task))) {
+            throw new IllegalArgumentException("Cannot create pull request for empty pull request title. Set it with git.pullRequestTitle property in configuration.");
+        }
+
+        if (isBlank(getMessage(task))) {
+            throw new IllegalArgumentException("Cannot create pull request for empty pull request description. Set it with git.pullRequestDescription property in configuration.");
+        }
+    }
+
     private String getMessage(CreatePullRequestTask task) {
-        return String.format("This pull request was automatically created by Shipkit's" +
-         " 'org.shipkit.upgrade-downstream' Gradle plugin (http://shipkit.org)." +
-        " Please merge it so that you are using fresh version of '%s' dependency.",
-            task.getVersionUpgrade().getDependencyName());
+        return task.getPullRequestDescription();
     }
 
     private String getTitle(CreatePullRequestTask task) {
-        UpgradeDependencyExtension versionUpgrade = task.getVersionUpgrade();
-        return String.format("Version of %s upgraded to %s", versionUpgrade.getDependencyName(), versionUpgrade.getNewVersion());
+        return task.getPullRequestTitle();
```

Comment:
It seems that these private methods getTitle and getMessage are no longer needed